app/src/main/java/com/example/workoutapp/ProfileActivity.kt

```diff
@@ -90,21 +92,21 @@ fun UserProfile(
             horizontalArrangement = Arrangement.SpaceBetween,
         ) {
             Column {
-                Text(text = "Height", fontWeight = FontWeight.Bold)
+                Text(text = stringResource(R.string.height), fontWeight = FontWeight.Bold)
```

Comment:
Using strings.xml is best practice and gets app ready for internationalization- https://developer.android.com/jetpack/compose/text

Reply:
Good.